docs/ruleset_functions/is_hvac_sys_fluid_loop_purchased_CHW.md

```diff
@@ -0,0 +1,27 @@
+# is_hvac_sys_fluid_loop_purchased_CHW  
+
+**Description:** Returns TRUE if the fluid loop associated with the cooling system associated with the HVAC system is attached to an external purchased chilled water loop. Returns FALSE if this is not the case.   
+
+**Inputs:**  
+- **B-RMR**: To evaluate if the fluid loop associated with the cooling system associated with the HVAC system is attached to an external purchased chilled water loop.   
+- **hvac_b.id**: The id of the hvac system to evaluate.  
+
+**Returns:**  
+- **is_hvac_sys_fluid_loop_purchased_CHW**: Returns TRUE if the fluid loop associated with the cooling system associated with the HVAC system is attached to an external purchased cooling loop. Returns FALSE if this is not the case.   
+ 
+**Function Call:** None  
+
+## Logic:   
+- Set is_hvac_sys_fluid_loop_purchased_CHW equal to FALSE: `is_hvac_sys_fluid_loop_purchased_CHW = FALSE`  
+- Check if RMR is not modeled with external fluid source: `if RMR.RuleModelInstance.external_fluid_source != Null:`  
```

Comment:
should this be Check if RMR is  modeled with external fluid source' (remove 'not')

Reply:
Good catch. Corrected.